.github/filters.yaml

```diff
@@ -0,0 +1,6 @@
+js:
```

Comment:
what about objc and swift?

Reply:
I only added filters for workflows in https://github.com/uber/piranha/tree/master/.github/workflows
It doesn't seem that we have CI for objc.